apps/test-suite/tests/FileSystemNext.ts

```diff
@@ -283,5 +283,63 @@ export async function test({ describe, expect, it, ...t }) {
         expect(output.path).toBe(file.path);
       });
     });
+
+    if (Platform.OS === 'ios') {
+      describe('App Group', () => {
+        const sharedContainerTestDir =
+          Directory.getSharedContainer('group.dev.expo.Payments').path + 'test/';
```

Comment:
I'm working on an api for exposing common paths like `cache` and `documents` here:
https://github.com/expo/expo/pull/31467/

maybe this should be a property on `Paths`, like `Paths.getSharedContainer()`

Reply:
_all file-system changes are reverted_